### DISCUSSION TITLE
How do I open it?

### DISCUSSION THREAD

**Discussion by AirplanegoBrr** [coder/code-server#3024] [Q&A] [answered]
I dont really understand how I would access it outside my network.(I know how to do the port forward on my wifi router) I think the password in my config file is ok so I dont wanna set up a lot more stuff. I dont know what the address to set in the config should be. Trying to access it VIA the iPad app on IOS if that helps!

**Comment by jsjoeio** [comment]
One thing you could do is run `code-server --link` and then accessing it via the browser on your iPad?

Here's a video explaining how to use link: [Deploying code-server with --link](https://www.youtube.com/watch?v=MdIEiZpg5vQ)

**Comment by nooobcoder** [comment, marked as answer]
To enjoy port forwarding from your router @AirplanegoBrr, **your ISP needs to allow that from their end too**. Be sure, that you **have a public IP address** provided by your ISP. Else, you can try better alternatives like `code-server --link` as provided by code-server itself. Else, you can also try, NGROK, Kite as other alternatives, but these are all exhaustible when under a free plan.

This is the default config.yaml file and it should work just fine.

++`bind-addr: 0.0.0.0:8880`

Please mark this discussion as answered, if I have answered your query.

**Reply by nooobcoder** [reply]
@AirplanegoBrr any update? Did this work out?

**Reply by AirplanegoBrr** [reply]
Yes! Thanks 😄

**Reply by nooobcoder** [reply]
@AirplanegoBrr please mark the discussion as answered, to make moderators aware of already answered discussions.

Thank you.

**Reply by jsjoeio** [reply]
Thanks for stepping in to help! @nooobcoder ♥️ We really appreciate it. I marked your discussion as the answer.

**Comment by davids-tips** [comment]
As of the issue of not being able to access it from the internet(outside network) without having to use ssh try setting up ngrok 

Hope this works for you,
> David A. Powell
> Beginner in Javascript
> Compentent in Python and Discord.py
> Proficient in html and css.
> OC of WhiskeyBot
> See my latest projects at https://Github.com/Davids-tips
> Founder of Whiskey Dev Group

**Reply by davids-tips** [reply]
Lol just noticed the guy above also said ngrok

**Reply by nooobcoder** [reply]
Yup, only downside, a `ngrok pro` account is needed to have a static link to the server.
